### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_ScreenReader:The selected file location is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6795]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version:1.10.0-insider (user setup)
Commit: 1ff9a2e
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Jaws version:2019.1907.42.ILM
NVDA version:2019.1.1

**Additional Details:**
Tools: Jaws, NVDA
MAS References:MAS1.3.1, MAS4.2.12

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select third radio button.
9. Navigate till next button & hit enter.
10. Start screen reader as nvda.
11. Navigate using tab key to the "..." button file location and select a file.

**Actual:**
On selecting a file, the newly added file location is not announced by the screen reader , the announcement is just the dialog and the button name.

**Similar issue is observed in jaws configuration.**

**Similar issue is observed on the following page:**
1.ExportDacpacSettings:
Follow the exact above steps but select radio button 2.

**Expected:**
The file location of the new file added should be announced as soon as it has been added in the edit box.
 
**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not know the location of the newly added file as soon as it is added.

**MAS Reference:**
MAS1.3.1:[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

MAS4.2.12:[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba})

**Attachment for Reference:**

**Comment by Benjin** [comment]
Questions added to doc:
Is this actually an issue given that the file was just announced in the other dialog?  If so…
How can we trigger narration of something irrespective of focus?  Would an acceptable fix be to manually set focus to the box with the newly-populated filepath?

**Comment by Benjin** [comment]
Note on this bug: NVDA appears to not handle the `aria-live` attribute correctly (rather, at all), so I've reproed and tested the fix using Narrator and JAWS

**Comment by fsteffi** [comment]
The issue still reproduces in the below version with jaws and nvda
Version: 1.12.0-insider
Commit: a584aca96997a18ecedee73a575536332e542718
Date: 2019-09-19T07:58:50.448Z
VS Code 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

[#6795 A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3686189/6795.A11y_AzureDataStudio.Schema.Dacpac.Backup._ImportBacpac_ImportBacpacSettings_ScreenReader.pptx)

**Comment by udeeshagautam** [comment]
Based on the discussion with Accessibility team over email - since this works on one free screen reader (Narrator) we will close this bug. @fsteffi please let us know if it doesnt work with Narrator.

**Comment by fsteffi** [comment]
@udeeshagautam as this application was an electron application, it was not tested with narrator. In the latest version the issue is still reproducing with jaws and NVDA.
Hence logging an external bug with nvda link: [https://github.com/nvaccess/nvda/issues/10365](https://github.com/nvaccess/nvda/issues/10365)

**Comment by udeeshagautam** [comment]
some work around like "with moving away and coming back to control should read the information" - should help the user in this case. Validate and add if not already there.
